Gems/AWSGameLift/Code/AWSGameLiftServer/Source/AWSGameLiftServerManager.cpp

```diff
@@ -416,6 +438,12 @@ namespace AWSGameLift
         return AZ::IO::Path();
     }
 
+    AZ::IO::Path AWSGameLiftServerManager::GetInternalSessionPrivateKey()
+    {
+        // GameLift doesn't support it, return empty path

```

Comment:
As this is for server to server communication, is there any chance we will overwrite private keys used for server to server communication uses case?

Reply:
not right now, even we need it later just add required implementation inside.